sw/device/lib/dif/dif_entropy_src.c

```diff
@@ -123,6 +123,63 @@ dif_result_t dif_entropy_src_fw_override_configure(
   return kDifOk;
 }
 
+dif_result_t dif_entropy_src_health_test_configure(
+    const dif_entropy_src_t *entropy_src,
+    dif_entropy_src_health_test_config_t config) {
+  if (entropy_src == NULL) {
+    return kDifBadArg;
+  }
+
+  ptrdiff_t high_thresholds_reg_offset = NULL;
+  ptrdiff_t low_thresholds_reg_offset = NULL;
```

Comment:
NULL is probably not a safe default offset, since the register could very well be at zero offset. Perhaps a negative value would work to indicate "none" here.

Reply:
fixed.